applications/socbayio.md

```diff
@@ -0,0 +1,112 @@
+
+Demo video got from ipfs.io/ipfs gateway
+![image](https://user-images.githubusercontent.com/16051365/115372829-94b11b80-a1cb-11eb-8ad7-f603bd364b93.png)
+
+Upload video
+![image](https://user-images.githubusercontent.com/16051365/115372758-8236e200-a1cb-11eb-80c9-b3820f308ff5.png)
+
+
+### Ecosystem Fit 
+
+* Where and how does your project fit into the ecosystem? 
+
+Sia.tech is a working platform for a long time but until the moment public people don't use it to store their files. Distributed storage in general doesn't have much use cases where people can trust and put their personal data on it. Socbay.io is the place where people can share their public videos and experience how easy and powerful a distributed storage system has. What people are worrying the most about when using distributed storage platforms is its privacy and its stability. Socbay.io can convince people about the stability by showing that their videos (in other word: their data) are always playable with high speed and low latency on Crust.Network.
+
+
+* Are there any other projects similar to yours in the IPFS/Crust ecosystem? 
+
+Lbry has their own video sharing platform https://odysee.com/ (previously lbry.tv) It's not fully distributed because the contents are guaranteed by private Lbry nodes. Public nodes don't have incentive to keep their nodes alive. Odysee has more than 15M visits each month. People start using distributed video sharing platform because they care more and more about the cencorship.
+
+D.tube is a distributed video sharing blockchain, this project is awesome but the weakest point of the project is that people should be responsable for the storage of their videos. This project could work for a certain amount of people but for public audience it doesn't work. I made a survey with my friends who are engineers in IT fields, they are not aware about distributed storage protocols and distributed storage platforms, so asking them to store videos themself without using centralized platforms is a diffucult mission.
+
+
+  * If so, how is your project different?
+  
+Socbay.io was built by Crust Network supporters therefore it's for Crust Network first. Socbay.io is fully decentralized and is maintained by donation and advertisement. 
+  
+
+## Team
+
+### Team members
+* Nguyen Trong Nghia
+
+### Contact
+* **Contact Name:** Nguyen Trong Nghia
+* **Contact Email:** trongnghia.nguyen@outlook.com
+
+### Legal Structure 
+* **No legal structure** 
+
+### Team Code Repos
+* https://github.com/nghianguyen119/
+
+### Team LinkedIn Profiles
+* https://www.linkedin.com/in/trongnghia/
+
+
+### Overview
+* **Total Estimated Duration:** 4 months
+* **Full-Time Equivalent (FTE):**  1
+* **Total Costs:** 15000 USD
+
+### Milestone 1: MVP website
+* **Estimated Duration:** 2 months
+* **FTE:**  1
+* **Costs:** 10,000 USD
+
+| Number | Deliverable | Specification |
```

Comment:
There is no Crust-related work in your first milestone which alone takes up 1w USD. We recommand that you integrate the feature of Crust storage order in your platform for the first milestone.

Reply:
Agree. I made the trial on this part and it's possible to integrate for the first milestone.